plaso/parsers/postgresql.py

```diff
@@ -0,0 +1,182 @@
+# -*- coding: utf-8 -*-
+"""Parser for PostgreSQL log files."""
```

Comment:
please add a note that this is multi lines log format

Reply:
Added a comment in the Parser docstring